### PR TITLE
Enabling CI and packaging

### PR DESCRIPTION
**Pull request by fossterer**
Primary goal: Fix 'Build: Failing' notifications by Travis CI
Additional Contributions:
    - .gitignore
    - Versioning

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+1.0.1
```
Comment (fossterer) [inline VERSION]: Semantic Versioning feels simple and natural to use

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    name = 'UnsplashUbuntu',
+    version = version,
+
+    description = 'Wallpaper Changer with multi-monitor support',
+    long_description = long_description,
+
+    url = 'https://github.com/PseudoAj/UnsplashUbuntu',
+
+    author = 'PseudoAj',
+    author_email = '',
```
Comment (fossterer) [inline setup.py:26]: It is up to you to fill this out

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+    license = 'GPL v3',
```
Comment (fossterer) [inline setup.py:28]: Not sure if there are any reservations; Using and persuading to give back in an open source fashion feels fine

Comment (PseudoAj) [reply setup.py:28]: I am good with GPL v3 @fossterer

**Comment by PseudoAj** [comment]
@fossterer Few questions regarding the packaging:
 - Is the packaging tested on ubuntu? If yes, please do let me know if it is 14.04.
 - Although travis is checking test cases with `pytest` there are no tests cases yet. Can we add test cases in our TODO?

P.S. Thanks for the contribution.

**Comment by fossterer** [comment]
- Yes, I tested it on Ubuntu 16.04.2 LTS
 - Since we have no testcases yet, I overrode the 'pytest' instruction. Travis is not running any. The task is already in TODO

**Comment by fossterer** [comment]
Going forward, I am considering managing the tasks originally in TODO as 'Issues' on Github

**Comment by PseudoAj** [comment]
That sounds great. Let's create issues and tag them.

**Pull request merged**